integration/tests/cook/test_cli.py

```diff
@@ -429,7 +429,7 @@ def test_show_running_job(self):
         self.assertEqual(0, cp.returncode, cp.stderr)
 
     def test_quoting(self):
-        cp, uuids = cli.submit('echo "Hello; exit 1"', self.cook_url)
+        cp, uuids = cli.submit('echo "Hello; exit 1"', self.cook_url, submit_flags=f'--max-retries 5')
```

Comment:
This change is unrelated to the rest of the PR, but the test failed on this branch, so I fixed this here.